messageforums-hbm/src/java/org/sakaiproject/component/app/messageforums/dao/hibernate/MessageImpl.hbm.xml

```diff
@@ -808,6 +808,17 @@
              where area.contextId = :contextId and
              message.deleted = false]]>
   </query>
+
+    <!-- returns recent forum  messages for a given siteId -->
+    <query name="findRecentDiscussionForumMessagesInSite">
+        <![CDATA[select message, topic, forum from org.sakaiproject.component.app.messageforums.dao.hibernate.MessageImpl as message
+                 join message.topic as topic
+             join topic.openForum as forum
+             join forum.area as area
+             where area.contextId = :contextId and
+             message.inReplyTo is null and
```

Comment:
Ok, but then the JavaDoc on the method is wrong as it doesn't get new messages it gets new threads.